flagd/Dockerfile

```diff
@@ -7,4 +8,8 @@ COPY flags/* .
 COPY scripts/* .
 LABEL org.opencontainers.image.source = "https://github.com/open-feature/test-harness"
 
-ENTRYPOINT ["sh", "wrapper.sh", "./flagd", "start", "-f", "file:testing-flags.json", "-f", "file:changing-flag.json"]
+ENTRYPOINT ["sh", "wrapper.sh", "./flagd", "start", \
+    "-f", "file:testing-flags.json", \
+    "-f", "file:changing-flag.json", \
+    "-f", "file:custom-ops.json", \
+    "-f", "file:zero-flags.json"]
```

Comment:
I've split the flags up into different files to make things easier. It also simplifies the "change" script